app/frontend/src/components/CircularProgress.tsx

```diff
@@ -6,9 +6,9 @@ import {
 import classNames from "classnames";
 import React, { ForwardedRef } from "react";
 
-const useStyles = makeStyles({
+const useStyles = makeStyles((theme) => ({
   root: {},
-});
+}));
```

Comment:
Should revert this

Reply:
Why do we need to keep this in then? 🤔

Reply:
We will definitely have a custom loading indicator with the logo